packages/cozy-client/src/CozyClient.js

```diff
@@ -340,17 +348,18 @@ class CozyClient {
       ? JSON.parse(root.dataset.cozy)
       : { ...root.dataset }
 
-    const { cozyDomain, cozyToken } = data
+    const { domain, token } = data
```

Comment:
Hmm, this one is a BC too. To have data.domain & data.token, we need to use `cozy-data={{ }}` in the app. This is not done everywhere yet. Else we have `data-cozy-domain="{{.Domain}}"` resulting of having data.cozyDomain .

Should we check both (cozyDomain && cozyToken) || (domain && token) ?

Reply:
Yep ! fortunately a test also caught the error :) I added support for both cozyToken and token and same for domain.